src/routes/testingOnly/index.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import reseedDB from './handlers';
+// import transactionWrapper from '../transactionWrapper';
+import testingOnly from '../../middleware/testingOnlyMiddleware';
+
+const router = express.Router();
+// must run outside of transaction wrapper as some of the action will run in a child process
+router.get('/reseed', testingOnly, reseedDB);
```

Comment:
do we want to put the access middleware on this?

Reply:
it is, thats the testingOnly middleware